apps/backend/src/modules/users/libs/types/user-password-details.type.ts

```diff
@@ -1,5 +1,12 @@
 type UserPasswordDetails = {
+	group: {
+		id: number;
+		key: string;
+		name?: string;
```

Comment:
name shouldn't be optional when group is present
```suggestion
		name: string;
```